pkg/ucp/backend/controller/resourcegroups/trackedresourceprocess_test.go

```diff
@@ -147,28 +144,6 @@ func Test_Run(t *testing.T) {
 		require.Equal(t, expected, result)
 		require.NoError(t, err)
 	})
-
-	t.Run("Failure (validate downstream: invalid downstream)", func(t *testing.T) {
```

Comment:
This is an error case that is no longer possible. We previously needed to guard against a misconfigured `radius` plane everywhere that UCP interacts it.